prompting/base/miner.py

```diff
@@ -219,3 +220,28 @@ def resync_metagraph(self):
 
         # Sync the metagraph.
         self.metagraph.sync(subtensor=self.subtensor)
+
+    def _forward(self, synapse: StreamPromptingSynapse) -> StreamPromptingSynapse:
```

Comment:
The `BaseStreamMinerNeuron._forward` doesn't need to await for anything. Since it calls the SomeMiner.forward() method (which is synchronous), there is no need for this to be async. 

However, the SomeMiner.forward method has an internal function that is async which is responsible for handling streaming.